templates/Package_summary.ss

```diff
@@ -1,4 +1,13 @@
 <a href="https://addons.silverstripe.org/add-ons/$Name.ATT" title="<%t SiteSummary.AddonsLinkTitle "View {package} on addons.silverstripe.org" package=$Title.ATT %>" class="package-summary__anchor" target="_blank" rel="noopener">
     <strong class="package-summary__title">$Title.XML</strong>
-    <% if $Description %><span class="package-summary__description">$Description.XML</span><% end_if %>
+    
+    <% if $Badges %>
+    <% loop $Badges %>
```

Comment:
We could just loop Badges right? At worst it'll be empty

Reply:
In theory, yes. There are issues in v3 with looping empty lists resulting in at least one pass of the internal HTML (with no variables filled in).
In at least... there were. Call it habit if has changed at some point.

Reply:
Sweet, tested. Guess it was fixed at some point in the past. Call it habit, now no longer needed (thank goodness!)